releng_sop/koji_block_package_in_release.py

```
@@ -110,17 +111,29 @@ def get_parser():
         default="default",
         help="Select environment in which the program will make changes.",
     )
+    parser.add_argument(
+        "-d", "--debug",
+        action="store_true",
+        help="Print traceback for exceptions. By default only exception messages are displayed.",
+    )
     return parser
 
 
 def main():
     """Main function."""
-    parser = get_parser()
-    args = parser.parse_args()
-    env = Environment(args.env)
-    release = Release(args.release_id)
-    clone = KojiBlockPackageInRelease(env, release, args.packages)
-    clone.run(commit=args.commit)
+    try:
+        parser = get_parser()
+        args = parser.parse_args()
+
+        env = Environment(args.env)
+        release = Release(args.release_id)
+        clone = KojiBlockPackageInRelease(env, release, args.packages)
+        clone.run(commit=args.commit)
+
+    except Error as e:
+        if not args.debug:
+            sys.tracebacklimit = 0
+        raise e
```

Plain 'raise' would be better to re-raise an exception.